src/ServerPrivate.cc

```diff
@@ -296,19 +241,11 @@ void ServerPrivate::AddRecordPlugin(const ServerConfig &_config)
               pluginElem->AddElementDescription(cPathElem);
               cPathElem = pluginElem->GetElement("compress_path");
               cPathElem->AddValue("string", "", false, "");
-              cPathElem->Set<std::string>(cmpPath);
+              cPathElem->Set<std::string>(_config.LogRecordCompressPath());
             }
 
             return;
           }
-
-          // If playback plugin also specified, do not add a record plugin
```

Comment:
I think this block is still needed, to make sure they don't specify both playback and record at the same time.

Reply:
Oh sorry you're right, I lost track of the ifs and thought this was inside `if (pluginName->GetAsString() == LoggingPlugin::RecordPluginName())`

Reply:
eec3984